docs/misc/vmdkops-admin-cli-spec.md

```diff
@@ -121,6 +121,18 @@ Show any interesting information about the service. This includes file paths of
 information, and PID of running service. A simple example is shown here, although it's possible
 that the exact format may be somewhat different.
 
+# Set
+Modify attribute settings on a given volume. The volume is identified by its full path on the datastore,
+for example if the volume name is `container-vol` then the volume is specified as "/vmfs/volumes/<datastore>/dockvols/container-vol.vmdk".
+The attributes to set/modify are specified as a comma separated list as "<attr1>=<value>, <attr2>=<value>....". For example,
+a command line would look like this.
+
+```
+$ vmdkops-admin set --volume=<full path of volume> --options="<attr1>=<value>, <attr2>=<value>, ..."
```

Comment:
This is the wrong place to update the user facing CLI documentation.. fixing it and posting a PR..